### PR TITLE
Move the 'get_datetime_diff' function to 'wazuh-testing' utils module

### LINKED ISSUE #2287: Move `get_datetime_diff` to wazuh_testing
## Description

As it was noted [here](https://github.com/wazuh/wazuh-qa/pull/2130#pullrequestreview-819638243), the following function should be moved to `wazuh_testing` instead of being defined inside a test, since it is generic enough:
https://github.com/wazuh/wazuh-qa/blob/45e74cbe3ef8e414dbcb29e7ffd6e91c568a1a98/tests/performance/test_cluster/test_cluster_performance/test_cluster_performance.py#L37-L44

The [cluster performance test](https://github.com/wazuh/wazuh-qa/blob/master/tests/performance/test_cluster/test_cluster_performance/test_cluster_performance.py#L37-L44) should also be updated so it imports the function after this change.

### PR DESCRIPTION
**Pull request by yanazaeva**
|Related issue|
|---|
|#2287|

## Description
This closes #2287. The aim of this pull is to move the `get_datetime_diff` function from the `cluster_performance_test` into the `wazuh_testing.utils` module. After this change, the test continues to work as expected:
```
(wazuh_qa_env) yanazaeva@pop-os:~/git/wazuh-qa/deps/wazuh_testing$ python3.9 -m pytest  ../../tests/performance/test_cluster/test_cluster_performance/ --artifacts_path=/docs/rc5/50000A25W --n_workers=25 --n_agents=50000
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.9.5, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /home/yanazaeva/git/wazuh-qa
plugins: testinfra-5.0.0, metadata-1.11.0, html-3.1.1
collected 1 item                                                                                                                                                                                                  

../../tests/performance/test_cluster/test_cluster_performance/test_cluster_performance.py .                                                                                                                 [100%]

================================================================================================ 1 passed in 0.50s ================================================================================================
```

||Results|
|:--:|:--:|
|R1| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8465374/1.zip) |
|R2| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8465376/2.zip) |
|R3| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8465377/3.zip) |

### CODE REVIEW

**Review by Selutario — DISMISSED** [review]

File: deps/wazuh_testing/wazuh_testing/tools/utils.py [hunk 1]
```diff
@@ -169,3 +168,13 @@ def format_ipv6_long(ipv6_address):
         str: IPV6 long form
     """
     return (ipaddress.ip_address(ipv6_address).exploded).upper()
+
+
+def get_datetime_diff(phase_datetimes, date_format):
+    """Calculate the difference between two datetimes.
+
+    Args:
+        phase_datetimes (list): List containing start and end datetimes.
```
Comment (juliamagan) [inline deps/wazuh_testing/wazuh_testing/tools/utils.py:177]: We need to add `date_format`

**Review by juliamagan — CHANGES_REQUESTED** [review]

File: tests/performance/test_cluster/test_cluster_performance/data/25w_50000a_thresholds.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ resources:
         workers:
           max: 47.85
           mean: 9.35
-          reg_cof: 0.06
+          reg_cof: 0.1
```
Comment (juliamagan) [inline tests/performance/test_cluster/test_cluster_performance/data/25w_50000a_thresholds.yaml:56]: Why is this change needed in this PR?

**Review by juliamagan — CHANGES_REQUESTED** [review]

File: tests/performance/test_cluster/test_cluster_performance/data/25w_50000a_thresholds.yaml (hunk 2 above)
Comment (Selutario) [reply tests/performance/test_cluster/test_cluster_performance/data/25w_50000a_thresholds.yaml:56]: The current thresholds are too small and strict, which causes most tests to be considered failed. It will be necessary to increase them little by little so that they only fail if an environment really has unusually high resource consumption.

**Comment by Selutario** [comment]
## PR update

I fixed the conflicts. These are the results of `test_cluster_performance.py` when using this [artifacts](https://github.com/wazuh/wazuh-qa/files/8762395/artifacts.zip):

```
python3 -m pytest tests/performance/test_cluster/test_cluster_performance/ --artifacts_path='/home/selu/Descargas/cluster_performance/107' --html=report.html --self-contained-html --n_workers=25 --n_agents=50000
============================================================================================ test session starts ============================================================================================
platform linux -- Python 3.8.10, pytest-5.0.0, py-1.8.2, pluggy-0.13.1
rootdir: /home/selu/Git/wazuh-qa
plugins: metadata-1.10.0, html-3.1.1, testinfra-5.0.0, tavern-1.2.2, pep8-1.0.6, cov-2.10.0, asyncio-0.14.0
collected 1 item                                                                                                                                                                                            

tests/performance/test_cluster/test_cluster_performance/test_cluster_performance.py .                                                                                                                 [100%]

---------------------------------------------------------------------- generated html file: file:///home/selu/Git/wazuh-qa/report.html ----------------------------------------------------------------------
=================================================================================== 1 passed, 2 warnings in 0.81 seconds ====================================================================================
```

**Comment by juliamagan** [comment]
## 24/05/2022

### Testing

We have used the artifacts attached [here](https://github.com/wazuh/wazuh-qa/pull/2782#issuecomment-1135825603)

||Results|
|:--:|:--:|
|R1| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8762576/R1-test-cluster-performance.zip)|
|R2| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8762579/R2-test-cluster-performance.zip) |
|R3| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8762580/R3-test-cluster-performance.zip)  |

**Review by juliamagan — APPROVED** [review]
LGTM
